[2019-11-23 02:24:20,063] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:24:20,213] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:24:20,215] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:27:17,949] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:27:18,069] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:27:18,077] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:27:21,135] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=42 HTTP/1.1" 200 324
[2019-11-23 02:27:22,438] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=40 HTTP/1.1" 200 327
[2019-11-23 02:27:25,747] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=40 HTTP/1.1" 200 327
[2019-11-23 02:27:26,644] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=42 HTTP/1.1" 200 324
[2019-11-23 02:27:27,770] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=40 HTTP/1.1" 200 327
[2019-11-23 02:27:31,101] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:27:31,403] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:27:31,535] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:30:41,605] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:30:41,780] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:30:41,791] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:30:48,389] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=40 HTTP/1.1" 200 327
[2019-11-23 02:30:50,578] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=42 HTTP/1.1" 200 324
[2019-11-23 02:30:52,085] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:31:07,907] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:34:30,320] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 02:34:30,482] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 02:34:30,935] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:34:31,166] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:34:31,166] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:34:35,512] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:35:21,891] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:35:22,013] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:35:22,017] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:35:24,616] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:39:07,729] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:39:07,892] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:39:07,915] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:39:11,201] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:42:39,116] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:42:39,351] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:42:39,361] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:42:40,289] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:45:04,198] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:45:04,338] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:45:04,356] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:45:07,878] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:45:40,939] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:45:41,057] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:45:41,067] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:45:42,724] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:48:29,491] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:48:29,642] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:48:29,677] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:48:32,593] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:50:04,971] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:50:05,105] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:50:05,123] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:50:06,932] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:51:39,693] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:51:39,863] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:51:39,894] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:51:42,250] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:52:22,167] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:52:22,319] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:52:22,339] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:52:25,487] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:53:21,838] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:53:21,970] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:53:21,989] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:53:26,015] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:54:43,955] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:54:44,100] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:54:44,104] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:54:46,678] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:55:25,244] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:55:25,391] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:55:25,401] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:55:27,570] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:56:38,943] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:56:39,078] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:56:39,106] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:56:40,905] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:56:54,139] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:56:54,344] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:56:54,344] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:56:57,839] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:58:19,194] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:58:19,339] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:58:19,347] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:58:21,529] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:59:13,929] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 02:59:14,044] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 02:59:14,062] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 02:59:16,452] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:00:27,475] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:00:27,656] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:00:27,657] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:00:29,344] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:00:36,828] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:00:36,988] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:00:37,019] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:00:45,559] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:01:07,028] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 62
[2019-11-23 03:02:42,694] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:02:42,872] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:02:42,880] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:02:45,633] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:02:55,383] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 62
[2019-11-23 03:04:36,440] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:04:36,612] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:04:36,653] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:04:40,186] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:07:00,566] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:07:00,716] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:07:00,725] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:07:03,468] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:08:46,947] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:08:47,131] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:08:47,134] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:08:49,226] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:08:51,910] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:08:52,033] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:08:53,592] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 62
[2019-11-23 03:09:00,116] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:09:00,276] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:09:00,299] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:09:06,732] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:09:06,888] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:09:06,918] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:09:09,892] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:09:17,807] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:09:17,972] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:09:51,178] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:09:51,378] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:09:51,381] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:11:57,611] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:11:57,771] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:11:57,781] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:12:02,879] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:12:06,114] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:12:06,249] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:12:09,816] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:12:10,033] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:12:10,042] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:12:56,590] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:12:56,727] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 498
[2019-11-23 03:12:56,734] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:13:03,778] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:13:21,353] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:13:21,696] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 516
[2019-11-23 03:13:40,478] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:13:40,690] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 516
[2019-11-23 03:14:04,398] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:14:04,646] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 522
[2019-11-23 03:14:12,442] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:14:12,647] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 528
[2019-11-23 03:14:29,164] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:14:29,365] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 673
[2019-11-23 03:14:45,049] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:14:45,254] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:14:48,348] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=40 HTTP/1.1" 200 528
[2019-11-23 03:14:50,079] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/users?page=1&roles=41 HTTP/1.1" 200 0
[2019-11-23 03:14:50,245] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=41 HTTP/1.1" 200 62
[2019-11-23 03:14:53,549] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=42 HTTP/1.1" 200 457
[2019-11-23 03:14:55,519] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:15:06,144] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:15:06,304] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:15:14,317] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:15:14,496] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:15:20,512] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:15:20,703] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 62
[2019-11-23 03:15:20,721] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:15:27,419] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:15:27,586] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:15:35,964] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:15:36,131] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:15:42,122] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/users HTTP/1.1" 200 13
[2019-11-23 03:15:42,372] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:15:45,726] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=42 HTTP/1.1" 200 312
[2019-11-23 03:15:49,492] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=41 HTTP/1.1" 200 298
[2019-11-23 03:15:51,237] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1&roles=40 HTTP/1.1" 200 528
[2019-11-23 03:15:52,550] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:17:04,108] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 0
[2019-11-23 03:17:05,169] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:17:05,180] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/ofroles HTTP/1.1" 200 0
[2019-11-23 03:17:05,183] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/users?page=1 HTTP/1.1" 200 0
[2019-11-23 03:17:05,355] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:17:05,362] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:17:08,359] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 0
[2019-11-23 03:17:08,911] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:17:08,921] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/type?pid=1 HTTP/1.1" 200 0
[2019-11-23 03:17:09,054] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 03:17:59,928] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:18:00,066] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:18:48,692] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:18:51,090] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:18:51,243] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:18:51,247] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:18:52,090] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:18:52,223] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:18:52,786] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:18:52,940] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 03:18:53,483] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:18:53,613] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:19:07,254] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:19:10,584] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:19:10,744] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:19:10,745] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:19:12,099] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:19:12,270] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 03:19:12,930] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:19:13,084] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:19:40,961] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:19:43,636] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:19:43,759] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:19:43,764] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:19:45,162] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:19:45,289] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 03:19:48,519] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:19:48,661] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:20:07,074] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:20:13,610] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:20:13,770] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:20:13,775] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:20:14,233] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:20:14,384] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 03:20:15,490] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:20:15,649] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:20:34,841] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:20:37,134] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:20:37,305] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:20:37,315] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:20:39,233] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:20:39,413] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 03:20:41,424] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:20:41,583] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:20:54,796] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:20:57,479] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:20:57,687] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:20:57,694] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:20:58,890] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:20:59,012] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 03:21:02,215] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:21:02,384] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:22:25,977] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:22:28,398] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:22:28,540] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:22:28,543] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:22:30,069] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:22:30,194] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 03:22:33,830] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:22:33,994] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 468
[2019-11-23 03:22:56,895] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/ofroles HTTP/1.1" 200 13
[2019-11-23 03:22:57,851] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 472
[2019-11-23 03:23:08,395] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/ofroles HTTP/1.1" 200 13
[2019-11-23 03:23:09,482] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 472
[2019-11-23 03:23:29,127] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4502
[2019-11-23 03:23:29,270] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 03:23:48,479] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction HTTP/1.1" 200 0
[2019-11-23 03:23:48,671] [basehttp.py:124] [basehttp:log_message] [INFO]- "DELETE /v1/base_dict/jurisdiction HTTP/1.1" 200 51
[2019-11-23 03:23:52,698] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:23:52,820] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 472
[2019-11-23 03:24:08,145] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/ofroles HTTP/1.1" 200 13
[2019-11-23 03:24:09,414] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 476
[2019-11-23 03:24:25,031] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/ofroles HTTP/1.1" 200 13
[2019-11-23 03:24:26,192] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 444
[2019-11-23 03:25:11,552] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/ofroles HTTP/1.1" 200 13
[2019-11-23 03:25:12,847] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 432
[2019-11-23 03:25:44,502] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:25:44,643] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 03:25:52,454] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:25:52,613] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:25:52,622] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 432
[2019-11-23 03:26:02,458] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:26:02,600] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 432
[2019-11-23 03:26:02,628] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:28:27,606] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:29:12,013] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:29:52,116] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:29:56,101] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:30:04,164] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:30:15,794] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:31:30,788] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 03:31:30,982] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 432
[2019-11-23 03:31:30,983] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 03:31:32,705] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:31:32,864] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 03:31:33,603] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:31:33,772] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 432
[2019-11-23 03:31:35,233] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:31:35,399] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 03:31:42,942] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:31:43,116] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 432
[2019-11-23 03:33:11,413] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 0
[2019-11-23 03:33:12,397] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:33:12,413] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/ofroles HTTP/1.1" 200 0
[2019-11-23 03:33:12,603] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 432
[2019-11-23 03:33:34,690] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 03:33:34,843] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 432
[2019-11-23 03:33:51,439] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/ofroles HTTP/1.1" 200 13
[2019-11-23 03:33:52,821] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 03:33:55,190] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/ofroles HTTP/1.1" 200 62
[2019-11-23 11:27:46,986] [basehttp.py:124] [basehttp:log_message] [INFO]- code 400, message Bad request syntax ('\x00¯\x02\x00\x0c\x00QUERY_STRING\x00\x00\x0e\x00REQUEST_METHOD\x03\x00GET\x0c\x00CONTENT_TYPE\x00\x00\x0e\x00CONTENT_LENGTH\x00\x00\x0b\x00REQUEST_URI\x01\x00/\t\x00PATH_INFO\x01\x00/\r\x00DOCUMENT_ROOT\r\x00/var/www/html\x0f\x00SERVER_PROTOCOL\x08\x00HTTP/1.1\x0e\x00REQUEST_SCHEME\x04\x00http\x0b\x00REMOTE_ADDR\r\x00192.168.0.101\x0b\x00REMOTE_PORT\x05\x0050939\x0b\x00SERVER_PORT\x02\x0080\x0b\x00SERVER_NAME\x01\x00_\t\x00HTTP_HOST\r\x00192.168.0.102\x0f\x00HTTP_CONNECTION')
[2019-11-23 11:27:47,018] [basehttp.py:124] [basehttp:log_message] [WARNING]- " ¯  QUERY_STRING   REQUEST_METHOD GET CONTENT_TYPE   CONTENT_LENGTH   REQUEST_URI /	 PATH_INFO / DOCUMENT_ROOT /var/www/html SERVER_PROTOCOL HTTP/1.1 REQUEST_SCHEME http REMOTE_ADDR 192.168.0.101 REMOTE_PORT 50939 SERVER_PORT 80 SERVER_NAME _	 HTTP_HOST 192.168.0.102 HTTP_CONNECTION" 400 -
[2019-11-23 11:27:47,385] [basehttp.py:124] [basehttp:log_message] [INFO]- code 400, message Bad request syntax ('\x00x\x02\x00\x0c\x00QUERY_STRING\x00\x00\x0e\x00REQUEST_METHOD\x03\x00GET\x0c\x00CONTENT_TYPE\x00\x00\x0e\x00CONTENT_LENGTH\x00\x00\x0b\x00REQUEST_URI\x0c\x00/favicon.ico\t\x00PATH_INFO\x0c\x00/favicon.ico\r\x00DOCUMENT_ROOT\r\x00/var/www/html\x0f\x00SERVER_PROTOCOL\x08\x00HTTP/1.1\x0e\x00REQUEST_SCHEME\x04\x00http\x0b\x00REMOTE_ADDR\r\x00192.168.0.101\x0b\x00REMOTE_PORT\x05\x0050939\x0b\x00SERVER_PORT\x02\x0080\x0b\x00SERVER_NAME\x01\x00_\t\x00HTTP_HOST\r\x00192.168.0.102\x0f\x00HTTP_CONNECTION')
[2019-11-23 11:27:47,386] [basehttp.py:124] [basehttp:log_message] [WARNING]- " x  QUERY_STRING   REQUEST_METHOD GET CONTENT_TYPE   CONTENT_LENGTH   REQUEST_URI /favicon.ico	 PATH_INFO /favicon.ico DOCUMENT_ROOT /var/www/html SERVER_PROTOCOL HTTP/1.1 REQUEST_SCHEME http REMOTE_ADDR 192.168.0.101 REMOTE_PORT 50939 SERVER_PORT 80 SERVER_NAME _	 HTTP_HOST 192.168.0.102 HTTP_CONNECTION" 400 -
[2019-11-23 11:28:30,068] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 0
[2019-11-23 11:28:32,565] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:28:33,427] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 0
[2019-11-23 11:28:34,384] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:31:14,480] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:31:23,032] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 0
[2019-11-23 11:31:23,258] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:31:23,270] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/ofroles HTTP/1.1" 200 0
[2019-11-23 11:31:23,271] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/users?page=1 HTTP/1.1" 200 0
[2019-11-23 11:31:23,468] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:31:23,468] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:31:28,141] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:31:28,200] [base.py:152] [base:get_response] [WARNING]- Not Found: /favicon.ico
[2019-11-23 11:31:28,201] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 2076
[2019-11-23 11:33:31,371] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/ HTTP/1.1" 200 7613
[2019-11-23 11:33:33,204] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/ HTTP/1.1" 200 7613
[2019-11-23 11:33:33,614] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 304 0
[2019-11-23 11:33:53,055] [basehttp.py:124] [basehttp:log_message] [INFO]- code 400, message Bad request syntax ('\x00¶\x02\x00\x0c\x00QUERY_STRING\x00\x00\x0e\x00REQUEST_METHOD\x03\x00GET\x0c\x00CONTENT_TYPE\x00\x00\x0e\x00CONTENT_LENGTH\x00\x00\x0b\x00REQUEST_URI\x07\x00/admin/\t\x00PATH_INFO\x07\x00/admin/\r\x00DOCUMENT_ROOT\r\x00/var/www/html\x0f\x00SERVER_PROTOCOL\x08\x00HTTP/1.1\x0e\x00REQUEST_SCHEME\x04\x00http\x0b\x00REMOTE_ADDR\r\x00192.168.0.102\x0b\x00REMOTE_PORT\x05\x0051982\x0b\x00SERVER_PORT\x02\x0080\x0b\x00SERVER_NAME\x01\x00_\t\x00HTTP_HOST\r\x00192.168.0.102\x0f\x00HTTP_CONNECTION')
[2019-11-23 11:33:53,055] [basehttp.py:124] [basehttp:log_message] [WARNING]- " ¶  QUERY_STRING   REQUEST_METHOD GET CONTENT_TYPE   CONTENT_LENGTH   REQUEST_URI /admin/	 PATH_INFO /admin/ DOCUMENT_ROOT /var/www/html SERVER_PROTOCOL HTTP/1.1 REQUEST_SCHEME http REMOTE_ADDR 192.168.0.102 REMOTE_PORT 51982 SERVER_PORT 80 SERVER_NAME _	 HTTP_HOST 192.168.0.102 HTTP_CONNECTION" 400 -
[2019-11-23 11:33:53,100] [basehttp.py:124] [basehttp:log_message] [INFO]- code 400, message Bad request syntax ('\x00y\x02\x00\x0c\x00QUERY_STRING\x00\x00\x0e\x00REQUEST_METHOD\x03\x00GET\x0c\x00CONTENT_TYPE\x00\x00\x0e\x00CONTENT_LENGTH\x00\x00\x0b\x00REQUEST_URI\x0c\x00/favicon.ico\t\x00PATH_INFO\x0c\x00/favicon.ico\r\x00DOCUMENT_ROOT\r\x00/var/www/html\x0f\x00SERVER_PROTOCOL\x08\x00HTTP/1.1\x0e\x00REQUEST_SCHEME\x04\x00http\x0b\x00REMOTE_ADDR\r\x00192.168.0.102\x0b\x00REMOTE_PORT\x05\x0051982\x0b\x00SERVER_PORT\x02\x0080\x0b\x00SERVER_NAME\x01\x00_\t\x00HTTP_HOST\r\x00192.168.0.102\x0f\x00HTTP_CONNECTION')
[2019-11-23 11:33:53,100] [basehttp.py:124] [basehttp:log_message] [WARNING]- " y  QUERY_STRING   REQUEST_METHOD GET CONTENT_TYPE   CONTENT_LENGTH   REQUEST_URI /favicon.ico	 PATH_INFO /favicon.ico DOCUMENT_ROOT /var/www/html SERVER_PROTOCOL HTTP/1.1 REQUEST_SCHEME http REMOTE_ADDR 192.168.0.102 REMOTE_PORT 51982 SERVER_PORT 80 SERVER_NAME _	 HTTP_HOST 192.168.0.102 HTTP_CONNECTION" 400 -
[2019-11-23 11:35:16,071] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:35:16,239] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /admin/ HTTP/1.1" 400 69551
[2019-11-23 11:35:16,397] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:35:16,442] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 400 69405
[2019-11-23 11:35:18,572] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:35:18,602] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /admin/ HTTP/1.1" 400 69551
[2019-11-23 11:35:18,707] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:35:18,748] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 400 69405
[2019-11-23 11:35:20,422] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:35:20,454] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /admin/ HTTP/1.1" 400 69551
[2019-11-23 11:35:20,564] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:35:20,613] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 400 69405
[2019-11-23 11:35:35,228] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:35:35,262] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /admin HTTP/1.1" 400 69545
[2019-11-23 11:35:36,957] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:35:36,992] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 400 69670
[2019-11-23 11:36:15,690] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:36:15,721] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /v1/base_dict/users?pase=1 HTTP/1.1" 400 69885
[2019-11-23 11:36:15,974] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:36:16,010] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 400 69429
[2019-11-23 11:36:21,693] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:36:21,728] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 400 69885
[2019-11-23 11:36:22,074] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:36:22,105] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 400 69429
[2019-11-23 11:37:10,860] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:37:10,896] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 400 69880
[2019-11-23 11:37:11,021] [exception.py:80] [exception:response_for_exception] [ERROR]- Invalid HTTP_HOST header: '192.168.0.102:8000'. You may need to add '192.168.0.102' to ALLOWED_HOSTS.
[2019-11-23 11:37:11,077] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 400 69424
[2019-11-23 11:38:09,928] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:38:09,980] [base.py:152] [base:get_response] [WARNING]- Not Found: /favicon.ico
[2019-11-23 11:38:09,980] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 2080
[2019-11-23 11:38:13,272] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:38:13,375] [base.py:152] [base:get_response] [WARNING]- Not Found: /favicon.ico
[2019-11-23 11:38:13,376] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 2080
[2019-11-23 11:45:43,441] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 0
[2019-11-23 11:45:43,548] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 0
[2019-11-23 11:45:44,026] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:45:44,149] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:45:52,278] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:45:52,292] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:46:06,391] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:46:06,627] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:46:08,816] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:46:08,958] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:46:12,947] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:46:17,304] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 0
[2019-11-23 11:46:17,472] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:46:17,484] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/ofroles HTTP/1.1" 200 0
[2019-11-23 11:46:17,485] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/users?page=1 HTTP/1.1" 200 0
[2019-11-23 11:46:17,645] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:46:17,647] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:46:56,423] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:46:56,515] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:47:09,436] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:47:09,616] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:47:39,092] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:47:39,327] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:47:47,926] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:47:48,092] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:47:59,713] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:47:59,816] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:48:08,715] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:48:08,870] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:48:50,549] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:48:50,740] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:48:51,406] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:48:54,070] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:48:54,185] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:48:54,199] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:49:06,675] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:49:16,057] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:49:16,096] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:49:16,944] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:49:17,501] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:49:18,366] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:49:20,652] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:49:20,782] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:49:20,790] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:49:27,384] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:49:27,533] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:49:29,293] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:49:29,414] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:49:29,438] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:49:37,812] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:50:15,342] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 0
[2019-11-23 11:50:15,621] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:50:15,645] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/ofroles HTTP/1.1" 200 0
[2019-11-23 11:50:15,645] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/users?page=1 HTTP/1.1" 200 0
[2019-11-23 11:50:15,804] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:50:15,806] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:50:25,761] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:50:25,941] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:51:29,331] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:51:29,386] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:51:30,573] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:51:32,850] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:51:33,105] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:51:33,106] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:51:51,810] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:51:51,926] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:51:52,442] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:51:52,780] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:51:54,118] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:51:56,167] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:51:56,312] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:51:56,313] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:51:58,238] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:51:58,249] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/type?pid=1 HTTP/1.1" 200 0
[2019-11-23 11:51:58,385] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 11:52:25,711] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:52:25,986] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:52:26,020] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:52:34,046] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:52:34,090] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/type?pid=1 HTTP/1.1" 200 0
[2019-11-23 11:52:34,361] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 11:52:34,713] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:52:34,856] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:52:34,865] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:52:39,714] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:52:39,872] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 11:53:09,985] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:53:10,140] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:53:10,146] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:53:14,139] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:53:14,619] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 11:53:23,956] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:53:24,162] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:53:24,167] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:53:26,965] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:54:26,322] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:54:26,482] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 11:54:44,079] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:54:44,215] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:54:44,222] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:55:27,932] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:55:28,236] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 11:56:15,326] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 0
[2019-11-23 11:56:15,839] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:56:16,245] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 11:56:22,637] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:56:22,785] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 11:56:28,141] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:56:28,289] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:56:28,298] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:56:44,984] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:56:47,028] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:56:47,155] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:56:47,163] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:57:00,589] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:57:00,676] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:57:01,279] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:57:03,838] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:57:03,996] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:57:03,996] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:57:05,082] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:57:05,234] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:57:10,411] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:57:10,546] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 11:57:10,570] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:57:12,592] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:57:12,775] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 11:57:20,668] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:57:20,830] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:57:36,796] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:57:36,974] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:58:38,171] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:58:38,314] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 11:59:03,610] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:59:03,759] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:59:05,249] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 11:59:05,507] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 11:59:08,907] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 11:59:09,068] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 11:59:09,079] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:00:49,881] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:00:50,076] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:00:50,087] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:01:31,404] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 12:01:31,415] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 12:01:37,709] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 12:01:37,743] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 12:01:44,165] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 12:01:44,339] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 12:01:50,338] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:01:50,598] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:01:50,619] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:01:52,227] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 12:01:52,375] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:01:52,950] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4364
[2019-11-23 12:01:53,083] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:02:06,222] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction HTTP/1.1" 200 0
[2019-11-23 12:02:07,322] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 200 51
[2019-11-23 12:02:21,966] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 200 51
[2019-11-23 12:02:27,697] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 200 51
[2019-11-23 12:02:32,710] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 200 51
[2019-11-23 12:02:40,725] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:05:40,508] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:05:40,750] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:06:29,001] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 0
[2019-11-23 12:06:29,456] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:06:29,808] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:07:02,571] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:07:03,193] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:08:36,137] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:08:36,877] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:09:24,616] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:09:24,866] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:09:58,702] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:09:58,945] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:10:07,085] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:10:07,735] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:10:34,287] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:10:34,920] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:10:59,321] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:10:59,835] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:13:08,254] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:13:08,405] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:13:08,408] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:18:01,447] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:18:01,614] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:18:01,636] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:18:03,476] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4373
[2019-11-23 12:18:03,745] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:18:23,004] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 200 51
[2019-11-23 12:18:43,185] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 200 51
[2019-11-23 12:19:05,020] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 200 51
[2019-11-23 12:19:20,671] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 200 51
[2019-11-23 12:19:22,820] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4509
[2019-11-23 12:20:13,124] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:20:13,509] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:20:13,521] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:20:15,099] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4509
[2019-11-23 12:20:15,272] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:20:53,704] [exception.py:135] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /v1/base_dict/jurisdiction
Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.IntegrityError: (1048, "Column 'created_time' cannot be null")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 100, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 174, in put
    sign=sign)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 394, in create
    obj.save(force_insert=True, using=self.db)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 837, in save_base
    self._save_parents(cls, using, update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 864, in _save_parents
    self._save_table(cls=parent, using=using, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 905, in _save_table
    forced_update)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 955, in _do_update
    return filtered._update(values) > 0
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 664, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1204, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 899, in execute_sql
    raise original_exception
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 889, in execute_sql
    cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.IntegrityError: (1048, "Column 'created_time' cannot be null")
[2019-11-23 12:20:54,012] [basehttp.py:124] [basehttp:log_message] [ERROR]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 500 253867
[2019-11-23 12:20:58,293] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:20:58,510] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:20:58,522] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:20:59,630] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4509
[2019-11-23 12:20:59,856] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:21:00,499] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4509
[2019-11-23 12:21:00,649] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:21:22,320] [exception.py:135] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /v1/base_dict/jurisdiction
Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.IntegrityError: (1048, "Column 'created_time' cannot be null")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 100, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 174, in put
    sign=sign)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 394, in create
    obj.save(force_insert=True, using=self.db)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 837, in save_base
    self._save_parents(cls, using, update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 864, in _save_parents
    self._save_table(cls=parent, using=using, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 905, in _save_table
    forced_update)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 955, in _do_update
    return filtered._update(values) > 0
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 664, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1204, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 899, in execute_sql
    raise original_exception
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 889, in execute_sql
    cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.IntegrityError: (1048, "Column 'created_time' cannot be null")
[2019-11-23 12:21:22,449] [basehttp.py:124] [basehttp:log_message] [ERROR]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 500 253867
[2019-11-23 12:21:24,241] [exception.py:135] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /v1/base_dict/jurisdiction
Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.IntegrityError: (1048, "Column 'created_time' cannot be null")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 100, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 174, in put
    sign=sign)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 394, in create
    obj.save(force_insert=True, using=self.db)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 837, in save_base
    self._save_parents(cls, using, update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 864, in _save_parents
    self._save_table(cls=parent, using=using, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 905, in _save_table
    forced_update)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 955, in _do_update
    return filtered._update(values) > 0
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 664, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1204, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 899, in execute_sql
    raise original_exception
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 889, in execute_sql
    cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.IntegrityError: (1048, "Column 'created_time' cannot be null")
[2019-11-23 12:21:24,379] [basehttp.py:124] [basehttp:log_message] [ERROR]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 500 253867
[2019-11-23 12:21:24,453] [exception.py:135] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /v1/base_dict/jurisdiction
Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.IntegrityError: (1048, "Column 'created_time' cannot be null")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 100, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 174, in put
    sign=sign)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 394, in create
    obj.save(force_insert=True, using=self.db)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 837, in save_base
    self._save_parents(cls, using, update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 864, in _save_parents
    self._save_table(cls=parent, using=using, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 905, in _save_table
    forced_update)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 955, in _do_update
    return filtered._update(values) > 0
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 664, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1204, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 899, in execute_sql
    raise original_exception
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 889, in execute_sql
    cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.IntegrityError: (1048, "Column 'created_time' cannot be null")
[2019-11-23 12:21:24,575] [basehttp.py:124] [basehttp:log_message] [ERROR]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 500 253867
[2019-11-23 12:21:24,625] [exception.py:135] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /v1/base_dict/jurisdiction
Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.IntegrityError: (1048, "Column 'created_time' cannot be null")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 100, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 174, in put
    sign=sign)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 394, in create
    obj.save(force_insert=True, using=self.db)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 837, in save_base
    self._save_parents(cls, using, update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 864, in _save_parents
    self._save_table(cls=parent, using=using, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 905, in _save_table
    forced_update)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 955, in _do_update
    return filtered._update(values) > 0
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 664, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1204, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 899, in execute_sql
    raise original_exception
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 889, in execute_sql
    cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.IntegrityError: (1048, "Column 'created_time' cannot be null")
[2019-11-23 12:21:24,794] [basehttp.py:124] [basehttp:log_message] [ERROR]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 500 253867
[2019-11-23 12:21:24,845] [exception.py:135] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /v1/base_dict/jurisdiction
Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.IntegrityError: (1048, "Column 'created_time' cannot be null")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 100, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 174, in put
    sign=sign)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 394, in create
    obj.save(force_insert=True, using=self.db)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 837, in save_base
    self._save_parents(cls, using, update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 864, in _save_parents
    self._save_table(cls=parent, using=using, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 905, in _save_table
    forced_update)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 955, in _do_update
    return filtered._update(values) > 0
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 664, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1204, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 899, in execute_sql
    raise original_exception
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 889, in execute_sql
    cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.IntegrityError: (1048, "Column 'created_time' cannot be null")
[2019-11-23 12:21:25,007] [basehttp.py:124] [basehttp:log_message] [ERROR]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 500 253867
[2019-11-23 12:21:25,115] [exception.py:135] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /v1/base_dict/jurisdiction
Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.IntegrityError: (1048, "Column 'created_time' cannot be null")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 100, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 174, in put
    sign=sign)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 394, in create
    obj.save(force_insert=True, using=self.db)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 837, in save_base
    self._save_parents(cls, using, update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 864, in _save_parents
    self._save_table(cls=parent, using=using, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 905, in _save_table
    forced_update)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 955, in _do_update
    return filtered._update(values) > 0
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 664, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1204, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 899, in execute_sql
    raise original_exception
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 889, in execute_sql
    cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.IntegrityError: (1048, "Column 'created_time' cannot be null")
[2019-11-23 12:21:25,288] [basehttp.py:124] [basehttp:log_message] [ERROR]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 500 253864
[2019-11-23 12:21:31,244] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4509
[2019-11-23 12:21:31,942] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4509
[2019-11-23 12:21:32,724] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4509
[2019-11-23 12:21:35,550] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:21:36,079] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4509
[2019-11-23 12:21:36,225] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:21:50,208] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:21:56,503] [base.py:152] [base:get_response] [WARNING]- Not Found: /admin
[2019-11-23 12:21:56,504] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /admin HTTP/1.1" 404 2062
[2019-11-23 12:22:06,754] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/ HTTP/1.1" 302 0
[2019-11-23 12:22:06,981] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/login/?next=/admin/ HTTP/1.1" 200 1652
[2019-11-23 12:22:07,017] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/css/login.css HTTP/1.1" 200 1203
[2019-11-23 12:22:07,017] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/css/base.css HTTP/1.1" 200 16066
[2019-11-23 12:22:07,026] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/css/fonts.css HTTP/1.1" 200 423
[2019-11-23 12:22:07,091] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 81348
[2019-11-23 12:22:07,093] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 80304
[2019-11-23 12:22:18,184] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[2019-11-23 12:22:18,381] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/ HTTP/1.1" 200 7613
[2019-11-23 12:22:18,395] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
[2019-11-23 12:22:18,415] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
[2019-11-23 12:22:18,415] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
[2019-11-23 12:22:18,416] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
[2019-11-23 12:22:18,440] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 82564
[2019-11-23 12:22:21,007] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/ HTTP/1.1" 200 24689
[2019-11-23 12:22:21,036] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/js/core.js HTTP/1.1" 200 7976
[2019-11-23 12:22:21,037] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/css/changelists.css HTTP/1.1" 200 6170
[2019-11-23 12:22:21,043] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/js/urlify.js HTTP/1.1" 200 8186
[2019-11-23 12:22:21,044] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 363
[2019-11-23 12:22:21,045] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/js/actions.js HTTP/1.1" 200 6501
[2019-11-23 12:22:21,060] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6591
[2019-11-23 12:22:21,063] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
[2019-11-23 12:22:21,064] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1538
[2019-11-23 12:22:21,085] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820
[2019-11-23 12:22:21,086] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 258648
[2019-11-23 12:22:21,203] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:22:21,265] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
[2019-11-23 12:22:23,701] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=3 HTTP/1.1" 200 25866
[2019-11-23 12:22:23,877] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:22:23,932] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097
[2019-11-23 12:22:26,913] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=2.3 HTTP/1.1" 200 26326
[2019-11-23 12:22:27,089] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:22:27,980] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3 HTTP/1.1" 200 26379
[2019-11-23 12:22:28,161] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:22:31,394] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=0 HTTP/1.1" 200 10577
[2019-11-23 12:22:31,576] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:22:32,911] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7484
[2019-11-23 12:22:33,154] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:22:35,390] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=0 HTTP/1.1" 200 10577
[2019-11-23 12:22:35,557] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:22:37,282] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7484
[2019-11-23 12:22:37,434] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:22:38,974] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=0 HTTP/1.1" 200 10577
[2019-11-23 12:22:39,179] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:22:56,384] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4509
[2019-11-23 12:22:58,689] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:22:59,032] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:22:59,301] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:23:00,517] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4509
[2019-11-23 12:23:00,656] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:23:26,573] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4509
[2019-11-23 12:23:28,799] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:23:29,732] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:23:29,734] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:23:30,250] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4509
[2019-11-23 12:23:30,437] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:23:49,143] [exception.py:135] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /v1/base_dict/jurisdiction
Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.IntegrityError: (1048, "Column 'created_time' cannot be null")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 100, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/tarena/桌面/git/Storage/server/storage/base_dict/views.py", line 174, in put
    sign=sign)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 394, in create
    obj.save(force_insert=True, using=self.db)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 837, in save_base
    self._save_parents(cls, using, update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 864, in _save_parents
    self._save_table(cls=parent, using=using, update_fields=update_fields)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 905, in _save_table
    forced_update)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/base.py", line 955, in _do_update
    return filtered._update(values) > 0
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/query.py", line 664, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1204, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 899, in execute_sql
    raise original_exception
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 889, in execute_sql
    cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/tarena/.local/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.6/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.6/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.IntegrityError: (1048, "Column 'created_time' cannot be null")
[2019-11-23 12:23:49,278] [basehttp.py:124] [basehttp:log_message] [ERROR]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 500 253854
[2019-11-23 12:25:00,305] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=0 HTTP/1.1" 200 10577
[2019-11-23 12:25:00,468] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:25:01,473] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=0 HTTP/1.1" 200 10577
[2019-11-23 12:25:01,705] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=0 HTTP/1.1" 200 10577
[2019-11-23 12:25:01,835] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=0 HTTP/1.1" 200 10577
[2019-11-23 12:25:02,103] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=0 HTTP/1.1" 200 10577
[2019-11-23 12:25:02,315] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=0 HTTP/1.1" 200 10577
[2019-11-23 12:25:02,485] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:25:05,281] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7484
[2019-11-23 12:25:05,423] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:25:06,520] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7484
[2019-11-23 12:25:06,665] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:25:07,058] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7484
[2019-11-23 12:25:07,269] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7484
[2019-11-23 12:25:07,452] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7484
[2019-11-23 12:25:07,580] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:25:07,864] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7484
[2019-11-23 12:25:08,034] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7484
[2019-11-23 12:25:08,210] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7484
[2019-11-23 12:25:08,346] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:25:10,635] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/add/?_changelist_filters=o%3D-2.3%26pid%3D7 HTTP/1.1" 200 8091
[2019-11-23 12:25:10,681] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/css/forms.css HTTP/1.1" 200 8173
[2019-11-23 12:25:10,682] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/js/change_form.js HTTP/1.1" 200 712
[2019-11-23 12:25:10,688] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495
[2019-11-23 12:25:10,699] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/css/widgets.css HTTP/1.1" 200 10340
[2019-11-23 12:25:10,891] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:25:42,811] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/base_dict/jurisdiction/add/?_changelist_filters=o%3D-2.3%26pid%3D7 HTTP/1.1" 200 8284
[2019-11-23 12:25:42,971] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:25:45,726] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/base_dict/jurisdiction/add/?_changelist_filters=o%3D-2.3%26pid%3D7 HTTP/1.1" 302 0
[2019-11-23 12:25:45,927] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7669
[2019-11-23 12:25:46,068] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:25:49,622] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7484
[2019-11-23 12:25:49,807] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:25:52,108] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3&pid=7 HTTP/1.1" 200 7484
[2019-11-23 12:25:52,259] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:25:57,308] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=-2.3 HTTP/1.1" 200 26956
[2019-11-23 12:25:57,458] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:26:03,585] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=1.-2.3 HTTP/1.1" 200 27367
[2019-11-23 12:26:03,778] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:26:12,845] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/48/change/?_changelist_filters=o%3D1.-2.3 HTTP/1.1" 200 8433
[2019-11-23 12:26:12,989] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:26:21,291] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/base_dict/jurisdiction/48/change/?_changelist_filters=o%3D1.-2.3 HTTP/1.1" 302 0
[2019-11-23 12:26:22,179] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=1.-2.3 HTTP/1.1" 200 27619
[2019-11-23 12:26:22,365] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:26:26,411] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4656
[2019-11-23 12:26:39,925] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=1.-2.3&pid=0 HTTP/1.1" 200 11019
[2019-11-23 12:26:40,259] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:26:42,109] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=1.-2.3&pid=1 HTTP/1.1" 200 7881
[2019-11-23 12:26:42,288] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:26:50,397] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=1.-2.3 HTTP/1.1" 200 27434
[2019-11-23 12:26:50,560] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:26:57,582] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/39/change/?_changelist_filters=o%3D1.-2.3 HTTP/1.1" 200 8455
[2019-11-23 12:26:57,738] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:27:01,902] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/base_dict/jurisdiction/39/change/?_changelist_filters=o%3D1.-2.3 HTTP/1.1" 200 8598
[2019-11-23 12:27:02,081] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:27:03,863] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/base_dict/jurisdiction/39/change/?_changelist_filters=o%3D1.-2.3 HTTP/1.1" 200 8598
[2019-11-23 12:27:04,086] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:27:07,826] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/base_dict/jurisdiction/39/change/?_changelist_filters=o%3D1.-2.3 HTTP/1.1" 302 0
[2019-11-23 12:27:08,813] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=1.-2.3 HTTP/1.1" 200 27549
[2019-11-23 12:27:09,008] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:27:10,963] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=1.-2.3 HTTP/1.1" 200 27364
[2019-11-23 12:27:11,134] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:27:16,017] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 0
[2019-11-23 12:27:16,786] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:27:19,022] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 0
[2019-11-23 12:27:19,212] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:27:19,224] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/ofroles HTTP/1.1" 200 0
[2019-11-23 12:27:19,232] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/users?page=1 HTTP/1.1" 200 0
[2019-11-23 12:27:19,418] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:27:19,418] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:27:27,351] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:27:43,766] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:27:53,448] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:29:35,695] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 0
[2019-11-23 12:29:36,076] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:29:36,262] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:29:36,553] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:30:05,297] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:30:05,316] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:30:05,877] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:30:08,649] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:31:02,529] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:31:02,787] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:31:03,481] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:31:11,806] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:31:11,892] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:31:12,304] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:31:17,022] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:32:55,016] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:32:55,070] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:32:55,999] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:32:57,742] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:33:27,075] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:33:27,150] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:33:27,896] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:33:30,012] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:34:05,289] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:34:05,430] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:34:06,022] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:34:09,022] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:35:37,056] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:35:37,105] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:35:37,882] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:35:40,105] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:37:18,269] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:40:11,158] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:47:05,676] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:47:59,798] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 0
[2019-11-23 12:48:00,983] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:48:01,005] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:48:01,631] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:48:02,143] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:48:45,166] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:48:45,172] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:48:46,194] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:02,535] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:02,650] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:03,103] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:03,784] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:03,864] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:04,359] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:06,986] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:07,004] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:07,526] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:09,487] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:09,500] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:10,918] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:12,004] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:12,018] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:12,862] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:12,896] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:13,641] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:16,801] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:16,818] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:17,655] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:17,693] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:18,273] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:18,896] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:31,726] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:31,867] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:32,326] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:33,080] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:33,414] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:34,039] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:34,483] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:41,997] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:42,136] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:42,923] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:46,451] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:49:46,619] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:49:46,620] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:49:48,927] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:49:49,173] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:49:59,524] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:49:59,722] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:49:59,745] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:50:01,543] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:50:01,797] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:50:16,340] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:50:16,666] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:50:16,666] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:50:17,395] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:50:17,543] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:50:17,544] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:51:01,631] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:51:01,791] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:51:02,552] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:51:08,901] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:51:08,931] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:51:09,453] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:58:12,523] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:58:12,711] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:58:12,757] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:58:14,176] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:58:14,184] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/type?pid=1 HTTP/1.1" 200 0
[2019-11-23 12:58:14,374] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:58:19,409] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:58:19,562] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:58:21,555] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:58:21,684] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:58:40,987] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/39/change/?_changelist_filters=o%3D1.-2.3 HTTP/1.1" 200 8465
[2019-11-23 12:58:41,163] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:58:47,772] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/base_dict/jurisdiction/39/change/?_changelist_filters=o%3D1.-2.3 HTTP/1.1" 302 0
[2019-11-23 12:58:48,644] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=1.-2.3 HTTP/1.1" 200 27618
[2019-11-23 12:58:48,800] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:58:55,981] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/39/change/?_changelist_filters=o%3D1.-2.3 HTTP/1.1" 200 8466
[2019-11-23 12:58:56,176] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:58:59,414] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/base_dict/jurisdiction/39/change/?_changelist_filters=o%3D1.-2.3 HTTP/1.1" 302 0
[2019-11-23 12:59:00,624] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=1.-2.3 HTTP/1.1" 200 27614
[2019-11-23 12:59:00,806] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:59:04,710] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/48/change/?_changelist_filters=o%3D1.-2.3 HTTP/1.1" 200 8435
[2019-11-23 12:59:05,092] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:59:08,981] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/base_dict/jurisdiction/48/change/?_changelist_filters=o%3D1.-2.3 HTTP/1.1" 302 0
[2019-11-23 12:59:09,717] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/base_dict/jurisdiction/?o=1.-2.3 HTTP/1.1" 200 27549
[2019-11-23 12:59:09,872] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7682
[2019-11-23 12:59:18,488] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 12:59:19,464] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 12:59:19,513] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 12:59:26,188] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4657
[2019-11-23 12:59:26,324] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 12:59:37,351] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 200 51
[2019-11-23 12:59:49,304] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 200 51
[2019-11-23 12:59:52,012] [basehttp.py:124] [basehttp:log_message] [INFO]- "PUT /v1/base_dict/jurisdiction HTTP/1.1" 200 51
[2019-11-23 12:59:54,159] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 12:59:54,276] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 13:00:04,115] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:00:17,760] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 13:00:17,913] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 13:00:17,914] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 13:00:20,586] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 13:00:20,755] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 13:00:20,758] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 13:00:21,996] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:00:22,148] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
[2019-11-23 13:00:56,713] [basehttp.py:124] [basehttp:log_message] [INFO]- "OPTIONS /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 0
[2019-11-23 13:00:57,245] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:00:57,263] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:00:57,852] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:00:59,718] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:01:15,101] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:01:15,277] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:01:15,811] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:01:16,218] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:01:19,247] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:01:54,007] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:01:54,094] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:01:54,784] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:01:55,280] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:03:01,580] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 13:03:01,697] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 13:03:01,700] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 13:03:04,046] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:03:07,788] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?pid=30 HTTP/1.1" 200 887
[2019-11-23 13:03:07,931] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/ofroles HTTP/1.1" 200 428
[2019-11-23 13:03:07,953] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/users?page=1 HTTP/1.1" 200 832
[2019-11-23 13:03:09,997] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/jurisdiction?type=1 HTTP/1.1" 200 4655
[2019-11-23 13:03:10,123] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /v1/base_dict/type?pid=1 HTTP/1.1" 200 248
